### DISCUSSION TITLE
utils.py throws an error on res2map stating string indices must be integer for e in value ['num']

### DISCUSSION THREAD

**Discussion by reshugupt1** [Valdes-Tresanco-MS/gmx_MMPBSA#399] [Q&A]
by running the command i am getting a error stating that string indices must be integer 
in miniconda3/envs/gmxmmpbsa/lib/python3.9/sitepackages/utils.py 
for e in value ['num']
can someone share the correct utils.py?
help will be appreciated

**Comment by marioernestovaldes** [comment]
This issue is usually related to inconsistencies in the group selection. Either you are selecting the wrong group in the index files or the groups are not in agreement to what is found in the topology
